worker-markup-core/src/main/java/com/github/cafdataprocessing/worker/markup/core/MarkupHeadersAndBody.java

```diff
@@ -154,10 +173,6 @@ private void markUpHeadersAndBody(final Element parentElement)
                         final String fullFromFieldValue = line + "\n" + splitFromFieldMatcher.group("SecondPartFromField");
                         addStandardisedHeader(nattyParser, headersElement, lines, fullFromFieldValue, ":\\*");
                     }
-                } // Only enter this block if we get a match i.e. line is "On xxx, abc wrote:"
-                else if (matcher.find()) {
-                    bodyIndex++;
-                    addCondensedHeader(nattyParser, headersElement, line, matcher);
```

Comment:
Let's discuss the move of this block - it looks like it changes the functionality.

Reply:
As discussed, let's move this in case there is a space after the "wrote:"